symbolic-debuginfo/src/pe.rs

```diff
@@ -516,4 +517,19 @@ impl<'data, 'object> PeEmbeddedPortablePDB<'data> {
         }
         Ok(output)
     }
+
+    /// Reads the Portable PDB contents into the provided file.
+    pub fn decompress_to_file(&self, output: File) -> Result<File, PeError> {
+        use std::io::prelude::*;
+        let buf_writer = BufWriter::new(output);
+        let mut decoder = flate2::write::DeflateDecoder::new(buf_writer);
```

Comment:
Wow, that is confusing, the same name in different modules.

But this looks interesting. We might as well just make this generic and have a `decompress_to<W>(...) where W: Write`
And you can just give that a `BufWriter<File>` or a `Vec`, both should be `Write`.

Reply:
Agreed, I've made the change